examples/storybook/src/custom-widgets.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, Inject, forwardRef } from '@angular/core';
+import { BaseWidget, NgAisInstantSearch } from 'angular-instantsearch';
+import { connectMenu } from 'instantsearch.js/es/connectors';
+
+@Component({
+  selector: 'ais-menu-select',
+  template: `
+    <select
+      class="menu-select"
+      (change)="state.refine($event.target.value)"
+    >
+      <option
+        *ngFor="let item of state.items"
+        [value]="item.value"
+        [selected]="item.isRefined"
+      >
+        {{item.label}}
+      </option>
+    </select>
+  `,
+})
+export class MenuSelect extends BaseWidget {
+  constructor(
+    @Inject(forwardRef(() => NgAisInstantSearch))
+    public instantSearchParent
+  ) {
+    super('MenuSelect');
+  }
+
+  public ngOnInit() {
+    this.createWidget(connectMenu, { attributeName: 'categories' });
+    super.ngOnInit();
+  }
+}
+
+@Component({
+  selector: 'ais-refresh',
+  template: `
+    <button
+      class="refresh"
+      (click)="refresh()"
+    >
+      refresh
+    </button>
+  `,
+})
+export class Refresh extends BaseWidget {
+  constructor(
+    @Inject(forwardRef(() => NgAisInstantSearch))
+    public instantSearchParent: any
+  ) {
+    super('Refresh');
```

Comment:
this one should implement the fake connector (probably in user code for now) as in #403

Reply:
Yes, can we fix that in the #403? I want to keep this PR (which already dense) fairly simple